README.md

```diff
@@ -1,3 +1,161 @@
 # Got Scraping
+Got Scraping is a small but powerful got extension that allows you to send browser-like requests with only a little configuration seamlessly and with a single function call. [Got](https://github.com/sindresorhus/got) is a widely use and powerful client that provides extensibility and customization. Sending browser-like requests is essential in the web scraping industry to blend in with the website traffic. You can read more about the mimics [here](#simulating-browser-like-requests).
 
-HTTP client based on well-known got library made for scraping purposes.
\ No newline at end of file
+## Motivation
+With the increasing popularity of web scraping, it is becoming more important to blend in with the typical internet traffic. This package implements multiple mimics to make GET requests similar to browsers. Thanks to the included [`header-generator`](https://github.com/apify/header-generator) package, you can choose various browsers from different operating systems and devices. The `header-generator` package generates all the headers for you so that you can focus on the important stuff.
+
+But the `got-scraping` doesn't stop here. The `got-scraping` package also tries to mimic the full request fingerprint, including the correct HTTP protocol and TLS ciphers.
+
+Proxies are essential in the web scraping industry. Another goal of this package is to simplify the usage of `HTTP` and `HTTPS` proxies. All you have to do is pass the `proxyUrl` option.
+
+There is one more good news for loyal `got` users. This package is modified got instance using handlers so that you can use the got interface as always. We've added just a few extras.
```

Comment:
I would probably wrap all `got`/`got-scraping` mentions with `` as otherwise it kinda blends in with the other text and makes we think some words are missing. But on the other hand it's rather subjective.

Reply:
I wanted to find a balance between these two options, but I guess it is better to be consistent than balanced 😃